extensions/mssql/src/objectManagement/interfaces.ts

```diff
@@ -285,6 +285,14 @@ export const enum UserType {
 	NoLoginAccess = 'NoLoginAccess'
 }
 
+/**
+ * Boolean options for boolean type dropdowns
+ */
+export enum BooleanDropdownOptions {
```

Comment:
@chlafreniere Do we usually need to localize boolean display values?

Reply:
If they're displayed as strings in the UI, then yes we should (I left another comment about why we aren't just using a checkbox here though)